File: app/src/main/java/com/vn/ezlearn/models/Banner.java
package com.vn.ezlearn.models;

/**
 * Created by FRAMGIA\nguyen.duc.manh on 15/09/2017.
 */

public class Banner {
    public static final int TYPE_URL = 1;
    public static final int TYPE_DRAWABLE = 2;
    public Integer id;
    public String imageUrl;
    public int imageDrawable;
    public int type;

    public Banner(Integer id, String imageUrl) {
        this.id = id;
        this.imageUrl = imageUrl;
        type = TYPE_URL;
    }
    public Banner(String imageUrl) {
        this.imageUrl = imageUrl;
        type = TYPE_URL;
    }
    public Banner(Integer id, int imageDrawable) {
        this.id = id;
        this.imageDrawable = imageDrawable;
        type = TYPE_DRAWABLE;
    }
}

File: app/src/main/java/com/vn/ezlearn/models/Banner.kt
package com.vn.ezlearn.models

/**
 * Created by FRAMGIA\nguyen.duc.manh on 15/09/2017.
 */

class Banner {
    var id: Int? = null
    var imageUrl: String = ""
    var imageDrawable: Int = 0
    var type: Int = 0

    constructor(id: Int?, imageUrl: String) {
        this.id = id
        this.imageUrl = imageUrl
        type = TYPE_URL
    }

    constructor(imageUrl: String) {
        this.imageUrl = imageUrl
        type = TYPE_URL
    }

    constructor(id: Int?, imageDrawable: Int) {
        this.id = id
        this.imageDrawable = imageDrawable
        type = TYPE_DRAWABLE
    }

    companion object {
        val TYPE_URL = 1
        val TYPE_DRAWABLE = 2
    }
}
